apps/src/netsim/NetSimBitLogPanel.html.ejs

```diff
@@ -1,5 +1,5 @@
 <%
-  var i18n = require('./locale');
+  var i18n = require('@cdo/netsim/locale');
```

Comment:
This is an approach to including locale files introduced by @pcardune in #9618 (and applied globally in #9621) which lets us stop worrying about require order for locale files in tests.  Where before we had to do this:

```javascript
var utils = require('../../util/testUtils.js');
// We must invoke setupLocales before requiring ModuleUnderTest,
// because it depends on the appropriate localization functions already
// being present on the blockly global.
utils.setupLocales();
var ModuleUnderTest = require('@cdo/apps/ModuleUnderTest');
```

Now we can just do this, because in tests the locale import _within_ `ModuleUnderTest` is aliased in tests to a version that does preloading if necessary.
```javascript
var ModuleUnderTest = require('@cdo/apps/ModuleUnderTest');
```

This was especially important as we move into ES6-land because Babel hoists `import` statements (but not `require` statements), foiling any attempt to do work before an import while using ES6 syntax.

This particular commit makes the same strategy work for NetSim files that don't depend on common locale, only netsim locale.  This wasn't causing failures on full test runs because the locale loading in other tests was setting up all locale files, but when using `karma:entry` to run NetSim tests by themselves (which I was doing while debugging the `_.debounce` issues) the locale wasn't being loaded.